docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,99 @@
+# Using Gambit with the Prover
+
+This is the mutation verifier which
+ checks that variants of the original
+ solidity program do not pass the specification.
+It uses mutations from the [Gambit](https://github.com/Certora/gambit)
+  mutation generator.
+It also allows users to include manually generated mutants.
+If a mutated program passes the specification,
+it may indicate that the specification is vacuous or not rigorous enough.
+In the rest of the document,
+  we refer to both the mutation generator and the verifier as Gambit.
+
+## Installations and Setup
+
+You will require java to run the mutation testing jar.
+Installation instructions can be found [here](https://www.java.com/en/download/help/download_options.html).
+
+## Running the Mutation Verifier
+
+- Example:
+```
+java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/Example.conf
+```
+
+- **NOTE: if a path has spaces, put quotes around it to ensure correct parsing by Kotlin's libraries**
+
+- Gambit supports `--staging`!
+
+**However, Gambit currently has trouble with
+`--send_only` and `--cloud` in the run scripts.
+If you have these flags, please remove them for now.**
+Apologies for the temporary inconvenience!
```

Comment:
so this is actually important. This means that the SE/SRs should not have these flags in their run scripts for now. I think this info should be visible as early as possible because in my experience, users have these flags most of the time.